interop/xds/server/server.go

```diff
@@ -0,0 +1,58 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Binary server for xDS interop tests.
+package main
+
+import (
+	"context"
+	"flag"
+	"net"
+	"os"
+	"strconv"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/grpclog"
+	testpb "google.golang.org/grpc/interop/grpc_testing"
+)
+
+var (
+	port        = flag.Int("port", 8080, "The server port")
+	serverID    = flag.String("server_id", "go_server", "Server ID included in response")
+	hostname, _ = os.Hostname()
```

Comment:
Optional: generate a random string if this fails, like the xds helloworld server: https://github.com/grpc/grpc-go/blob/master/examples/features/xds/server/main.go#L64
This would need to be done for all languages.